src/components/GirderLocationChooser.vue

```diff
@@ -12,6 +12,7 @@
           class="pl-4"
           root-location-disabled
           :location="selected"
+          :readonly="true"
```

Comment:
can't you go with only "readonly" property instead of ":readonly="true""?

Reply:
Yes you're right!